pkg/profiler/cpu/cpu.go

```diff
@@ -57,7 +57,7 @@ const (
 
 	defaultRLimit = 1024 << 20 // ~1GB
 
-	programName = "profile_cpu"
+	programName = "do_sample"
```

Comment:
:see_no_evil: I think I forgot to rename the function in bpf program while rebasing.

Reply:
Let's rename the function in the C file.

Reply:
Sounds good! Perhaps a name like `take_sample` is more representative of what the function is doing as the "profile CPU" bit happens on the perf event side?

Reply:
Sounds good to me!